### PR TITLE
Unit Test Setup

### PR DESCRIPTION
**Pull request by alfiehanssen**
This branch adds a test target and runs tests on build.

### CODE REVIEW

**Comment by alfiehanssen** [comment]
@huebnerob @ghking Same here.

File: VIMUploadTests/VIMUploadTests.m [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+#import <XCTest/XCTest.h>
+
+@interface VIMUploadTests : XCTestCase
+
+@end
+
+@implementation VIMUploadTests
+
+- (void)setUp {
```
Comment (ghking) [inline VIMUploadTests/VIMUploadTests.m:17]: new lines for brackets even though I know these were prob generated

Comment (alfiehanssen) [inline VIMUploadTests/VIMUploadTests.m:17]: Good catch, I'll add to all once I write the first unit test...

**Comment by huebnerob** [comment]
:+1:

**Pull request merged**